email_functions.py

```diff
@@ -1,42 +1,30 @@
-'''
-reading_mail() - takes no input and returns a dictonary with mail argumnets as dict['subject']  , dict['msg_body']
-send_mail() - takes 2 arguments - subject , message and returns 1 if message is sent successfully and 0 is a error occurs
-'''
-import gmail
-import smtplib
-import json
-from email.mime.text import MIMEText
+# email_functions.py
+
 import Logger
 import os
+import requests
 
 logger = Logger.Logger(name='RunLog')
-f_email = os.getenv('SENDER_EMAIL_ID')
-t_email = os.getenv('SENDEE_EMAIL_ID')
-passwd = os.getenv('EMAIL_PASS')
-
-
-def reading_mail () :  # this function returns a dictionary with email arguments
-	g = gmail.Gmail()
-	g.login(f_email, passwd)  #logging in to gmail server
-	unread = g.inbox().mail(unread=True)  #getting all unread mails. It returns all the blank mials
-	unread[-1].fetch()   # getting all the mail attributes like body,subject etc
-	mail_args = {'subject' : unread[-1].subject , 'msg_body' : unread[-1].body}
-	unread[-1].read()  #marking the mail as read
-	return mail_args
-	g.logout()  #logging out
+key = os.getenv('MAILGUN_API_KEY')
+recipient = os.getenv('SENDEE_EMAIL_ID')
```

Comment:
`SENDEE` is cute, but it's also confusing. Better to use `SENDER` and `RECIPIENT` as the standard notation here. :slightly_smiling_face: